### PR TITLE
feat: added user submissions

### PR DESCRIPTION
**Pull request by chetanguptaa**
#5

### CODE REVIEW

File: src/components/ui/input.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
+  ({ className, type, ...props }, ref) => {
+    return (
+      <input
+        type={type}
+        className={cn(
```
Comment (hkirat) [inline src/components/ui/input.tsx:13]: are these coming from shadcn? They should already be present in the `shad` folder

**Comment by hkirat** [comment]
Can u post some screenshots of what you've built here?

**Comment by chetanguptaa** [comment]
Screenshot (51)

File: src/components/ui/input.tsx (hunk 1 above)
Comment (chetanguptaa) [reply src/components/ui/input.tsx:13]: yes these are coming from shadcn

Comment (chetanguptaa) [reply src/components/ui/input.tsx:13]: i thought the shad files were in the ui folder in components

**Comment by hkirat** [comment]
hmm let me figure out the UX of this before committing to it.

**Comment by chetanguptaa** [comment]
> hmm let me figure out the UX of this before committing to it.

alright np

**Pull request closed without merging**